benchmarks/src/main/scala/zio/GenBenchmarks.scala

```diff
@@ -0,0 +1,81 @@
+package zio
+
+import java.util.concurrent.TimeUnit
+
+import hedgehog.core.GenT
+import org.openjdk.jmh.annotations._
+import org.scalacheck
+import zio.random.Random
+import zio.test.Gen
+
+@State(Scope.Thread)
+@BenchmarkMode(Array(Mode.AverageTime))
+@OutputTimeUnit(TimeUnit.NANOSECONDS)
+class GenBenchmarks {
+
+  @Param(Array("1000"))
+  var size: Int = _
+
+  @Benchmark
+  def zioDouble: Gen[Random, Double] =
+    Gen.double(0, size)
```

Comment:
Things that just return `Gen` aren't useful to benchmark. You should use the `Gen` to generate a whole bunch of numbers (e.g. `size`.numbers).